core/field_variable.js

```diff
@@ -316,6 +316,8 @@ Blockly.FieldVariable.dropdownCreate = function() {
     // Set the UUID as the internal representation of the variable.
     options[i] = [variableModelList[i].name, variableModelList[i].getId()];
   }
+  options.push('SEPARATOR');
+  options.push([Blockly.Msg.NEW_VARIABLE, 'CREATE_VARIABLE']);
```

Comment:
Consider adding a constant (Blockly.CREATE_VARIABLE_ID?) instead of using a string directly here.

Reply:
Yeah that makes sense. I was just being lazy haha. FYI https://github.com/google/blockly/blob/master/core/variables.js#L154 doesn't use a constant either.

Reply:
I'll fix that :)